.pre-commit-config.yaml

```diff
@@ -74,7 +74,6 @@ repos:
     hooks:
     -   id: python-no-eval
     -   id: python-no-log-warn
-    -   id: python-use-type-annotations
```

Comment:
Any reason you're killing type annotation checking in this repo?

Reply:
The code needs to be compatible with Python 2, which requires type annotations to be in comments.
This hook wants type annotations to be Python 3 style.